### PR TITLE
created skeleton, mysql schema. Implemented all data for component an…

### PR DESCRIPTION
**Pull request by Malyarc**
…d seeding file. Server/DB/Client all connect and works

### CODE REVIEW

File: database/schema.sql [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+CREATE TABLE colors (
+  id INTEGER AUTO_INCREMENT,
+  product_id INTEGER NOT NULL,
+  color_name TEXT,
+  color_url TEXT,
+  PRIMARY KEY (ID)
+);
+
+INSERT INTO colors (id, product_id, color_name, color_url) VALUES (1, 1, "Black", "https://hrr45fec.s3-us-west-1.amazonaws.com/mask_colors/il_1588xN.2290355171_9ptb.jpg");
```
Comment (ikheraj17) [inline database/schema.sql:16]: Very easy to read insertion of our 1st product pictures! lgtm!

File: database/seed.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+connection.connect();
+
+for (var i = 9; i < 100; i++) {
+  var productid = (Math.floor(Math.random() * 9) + 2);
+  var colorName = faker.commerce.color();
+  var colorURL = faker.image.imageUrl();
+
+  var queryString = `INSERT INTO colors (product_id, color_name, color_url) VALUES (${productid} , "${colorName}", "${colorURL}");`;
+
```
Comment (ikheraj17) [inline database/seed.js:20]: Maybe handle errors with the callback function here? Otherwise lgtm!

Comment (wsamsnyder) [reply database/seed.js:20]: Should the `colorURL`  be referencing a picture's URL that is stored in your S3 bucket?

Comment (mta630) [reply database/seed.js:20]: Ya I think you will need to change that faker.image.imageUrl() to grab pictures from an array of URLS from your s3, I think you were planning on doing that anyhow. Also I agree that you should have some sort of error handling for your query.

File: package.json [hunk 3]
```diff
@@ -8,10 +8,27 @@
     "node": ">=6.13.0"
   },
   "scripts": {
+    "react-dev": "webpack -d --watch",
+    "server-dev": "nodemon server/server.js"
```
Comment (mta630) [inline package.json]: You'll need to create a script here to run the seed file

File: server/server.js [hunk 4]
```diff
@@ -1 +1,27 @@
...
+const express = require('express');
+let app = express();
+
+const bodyParser = require('body-parser');
+
+app.use(express.static(__dirname + '/../client/dist'));
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
```
Comment (mta630) [inline server/server.js:10]: I am just curious what your reason behind using urlencoded is here?

**Review by mta630 — CHANGES_REQUESTED** [review]
It mostly looks good but i'm curious about the urlencode as well as I think you should have error handling in your seeding file and a script to run the seed file

File: database/seed.js (hunk 2 above)
Comment (Malyarc) [reply database/seed.js:19]: Hmm to clarify @Sam do we need our dummy data to have real imgs from S3 as well? 

So the page you are looking at is just populating the dummy data. I inserted the 10 real ones to display inside my Schema.sql file so I could source it. Sorry for not clarifying earlier

**Comment by Joseph765** [comment]
lgtm other than what micheal pointed out!

**Review by mta630 — APPROVED** [review]

File: package.json [hunk 5]
```diff
@@ -8,10 +8,28 @@
     "node": ">=6.13.0"
   },
   "scripts": {
+    "react-dev": "webpack -d --watch",
+    "server-dev": "nodemon server/server.js",
+    "seed": "node database/seed.js"
```
Comment (Joseph765) [inline package.json:13]: Looks good and you added a seeding script!

**Review by Joseph765 — COMMENTED** [review]
Great job!

**Review by Joseph765 — APPROVED** [review]
I approve!

**Review by mta630 — APPROVED** [review]
Looks great! approved

**Pull request merged**